src/components/dashboard/menu-table/menu-table.js

```diff
@@ -263,6 +288,16 @@ const MenuTable = (props) => {
             <MenuControls>
                 <form onSubmit={handleSearch} className='searchForm'>
                     <input className='search' name='search' placeholder='Search Dishes...' id='searchBox' type='text' />
+                    { 
+                        isSearching ? 
+                        <input className='cancelSearch' type='image' alt="Reset search" src={CancelIcon} onClick={(e) => { 
+                            e.preventDefault();
+                            document.getElementById('searchBox').value ='';
```

Comment:
Did not look at this part when it's first submitted but is it possible to use a state ://